doc/programming_guide/asr_ports.rst

```diff
@@ -4,7 +4,7 @@
 Speech Recognition Ports
 ########################
 
-A ports of the Sensory speech recognition library is provided.
+A ports of the Sensory and Cyberon speech recognition libraries is provided.
```

Comment:
`Ports` at the start and `are provided` at the end.

Have the full names of these libraries (eg DSpotter, etc) been defined earlier in the doc - if not then they should be added here.

Reply:
The first time the libraries are mentioned is in https://github.com/lucianomartin/sln_voice/blob/feature/Cyberon/doc/programming_guide/ffd/overview.rst. Is that ok or we should give more details?